### DISCUSSION TITLE
What to else to throw at Puma to make it process requests faster?

### DISCUSSION THREAD

**Discussion by jeffsaremi** [puma/puma#3144] [General]
We have maxed the workers. Set threads to 6. Is there anything I can do with additional memory? Are there internal hard-coded stuff what would limit this application from using additional memory (to hopefully process requests faster)?

**Comment by nateberkopec** [comment]
If you have Ruby 3.2+, you can turn on YJIT. That will consume maybe 20% additional memory but 10% faster latency.

There's nothing _Puma_ can do to decrease latency.

**Reply by jeffsaremi** [reply]
Thanks Nate! Not there yet
